contributing.md

```diff
@@ -100,6 +100,12 @@ Before starting major work, you should touch base with the maintainers of Featur
   # view docs locally
   open build/html/index.html
   ```
+* Before you commit, a few lint fixing hooks will run. You can also manually run these.
+  ```bash
+  pip install pre-commit
+  pre-commit install
```

Comment:
Do we need to list the `pip install pre-commit` and `pre-commit install` commands or will that be covered when users run `make installdeps`?

I created a fresh environment and just ran `make installdeps` and it seems like everything worked.